apps/ai-image-tagging/frontend/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ai-image-tagging-frontend",
+  "version": "1.0.0",
+  "private": true,
+  "main": "build/index.html",
+  "devDependencies": {
+    "@babel/core": "^7.8.4",
+    "@babel/plugin-proposal-class-properties": "^7.3.4",
+    "@babel/plugin-transform-runtime": "^7.3.4",
+    "@babel/preset-env": "7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "@contentful/contentful-extension-scripts": "0.14.1",
+    "@contentful/eslint-config-extension": "0.2.0",
+    "@testing-library/react": "8.0.4",
+    "contentful-cli": "0.33.2",
+    "cssnano": "4.1.10",
+    "eslint": "^6.0.1"
+  },
+  "scripts": {
+    "start": "contentful-extension-scripts start --serveOnly",
+    "build": "contentful-extension-scripts build",
+    "lint": "eslint ./ --ext .js,.jsx,.ts,.tsx",
+    "test": "contentful-extension-scripts test --env=jsdom --watch",
+    "test:coverage": "contentful-extension-scripts test --env=jsdom --coverage",
+    "configure": "contentful space use && contentful space environment use",
+    "login": "contentful login",
+    "logout": "contentful logout",
+    "help": "contentful-extension-scripts help"
+  },
+  "dependencies": {
+    "@contentful/forma-36-fcss": "^0.0.20",
+    "@contentful/forma-36-react-components": "^3.11.3",
+    "@contentful/forma-36-tokens": "^0.3.0",
+    "contentful-ui-extensions-sdk": "3.9.0",
```

Comment:
This might need to be updated as we had a few issues yesterday with it. I have it locally as `^3.10.7`

Reply:
This you can update in your branch, frontend in this PR is just a generated scaffolding